language/fr/declaration.lang

```diff
@@ -1,5 +1,38 @@
 <?php
 
-include("{$_EnginePath}language/en/declaration.lang");
+$_Lang['Title']                             = 'Déclaration IP partagée';
+
+$_Lang['DeclarationInfo']                   = 'Saisissez les surnoms de tous les joueurs qui partagent un ordinateur, une adresse IP ou qui se connectent occasionnellement au jeu aux mêmes endroits que vous (par exemple, le réseau de votre domicile, la même école, le même lieu de travail). <br/> Séparé tous les surnoms utilisant des virgules. Vous n\'avez pas besoin d\' ajouter votre propre surnom.';
+
+$_Lang['DeclarationType']                   = 'Raison';
+$_Lang['OnePC']                             = 'Ordinateur Partagé';
+$_Lang['OneIP']                             = 'Connexion Réseau Partagée';
+$_Lang['SamePlaces']                        = 'MCOnnexion dans un même lieux (école, lieu de travail, etc.)';
```

Comment:
`MCOnnexion` word looks like a typo, do you need to change that, or is it actually fine?